src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <React.Fragment>
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age && <p className="Person__age">I am {person.age}</p>}
+      {person.isMarried === true && person.sex === 'm' && (
+        <p className="Person__partner">{person.partnerName} is my wife</p>
+      )}
+      {person.isMarried === true && person.sex === 'f' && (
+        <p className="Person__partner">{person.partnerName} is my husband</p>
+      )}
+      {person.isMarried === false && (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
Try to simplify this logic and move it to helper variables and use it here
```suggestion
      {person.isMarried && person.sex === 'm' && (
        <p className="Person__partner">{person.partnerName} is my wife</p>
      )}
      {person.isMarried && person.sex === 'f' && (
        <p className="Person__partner">{person.partnerName} is my husband</p>
      )}
      {!person.isMarried && (
        <p className="Person__partner">I am not married</p>
      )}
```